browser/ui/views/crash_report_permission_ask_dialog_view.cc

```diff
@@ -231,6 +237,9 @@ void CrashReportPermissionAskDialogView::OnAcceptButtonClicked() {
 }
 
 void CrashReportPermissionAskDialogView::OnWindowClosing() {
+  g_browser_process->local_state()->SetBoolean(
+      kAskCrashReportPermission, !dont_ask_again_checkbox_->GetChecked());
```

Comment:
I won’t hold up the PR on this, but I don’t love the double negative. The permission could be called kDontAskForCrashReporting defaulted false.

Reply:
Yeah, it's much better. Thanks for suggestion!
fixed by https://github.com/brave/brave-core/pull/10093/commits/020fb73d0126c76357ad9310caeb5ef567674f46

Reply:
Thanks for fixing @simonhong 😄 Always easier to fix now - love seeing consistent usage for naming